site/docs/concepts/schemas.md

```diff
@@ -312,6 +351,29 @@ oneOf:
 # [1, 2], [3, 4, 5], [] => []
 ```
 
-Combining schema conditionals with annotations can be used to build
+You can combine schema conditionals with annotations to build
 [rich behaviors](../reference/reduction-strategies/composing-with-conditionals.md).
 
+## `default` annotations
+
+You can use `default` annotations to prevent null values from being materialized to your endpoint system.
+
+When this annotation is absent for a non-required field, missing values in that field are materialized as `null`.
+When the annotation is present, missing values are materialized with the field's `default` value:
+
+```yaml
+collections:
+  acmeCo/coyotes:
+    schema:
+      type: object
+      required: [id]
+      properties:
+        id: {type: integer}
+        anvils_dropped: {type: integer}
+          reduce: {strategy: sum }
+          default: 0
```

Comment:
This seems like a good example. It could get more complicated than this but for the vast majority of users I think they would be using defaults in exactly this way so I think its good to keep things easy to understand.